falco/CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 This file documents all notable changes to Falco Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
 
+## v1.19.0
+
+* Upgrade to Falco 0.32.0 (see the [Falco changelog](https://github.com/falcosecurity/falco/blob/0.32.0/CHANGELOG.md))
+* Various Falco config settings were updated for Falco 0.32.0
+
+### Breaking Changes
+
+* Audit Log is now supported via k8saudit plugin (when enabled, syscall instrumentation will be disabled)
+* dynamicBackend support for Audit Log is now deprecated
+* Rules files are not shipped with the chart anymore (Falco will use the rules files shipped with the container image)
```

Comment:
@leogr 
According to the https://github.com/falcosecurity/charts/pull/340#issuecomment-1148529350, should we remove this line from the Breaking Changes section? I think it might cause confusion.

Apart from that, why did you revert the commit? I just wanted to know the background for your decision.

Reply:
Hey @toVersus 

You are right. I will fix it. Thanks for noticing!

The decision was just conservative. @Issif just pointed out that having rule files in a config map makes life easier when one is debugging or just for just editing on the fly. So I just decided to revert the commit, hoping to find a better solution later.

Reply:
:point_right: https://github.com/falcosecurity/charts/pull/344